files/Aboot/boot0.j2

```diff
@@ -86,7 +86,7 @@ installer_image_path="$image_path/$installer_image"
 
 boot_config="$target_path/boot-config"
 
-cmdline_allowlist="crashkernel hwaddr_ma1"
+cmdline_allowlist="crashkernel hwaddr_ma1 sonic_fips fips"
```

Comment:
>sonic_fips fips [](http://example.com/codeflow?start=42&length=15)

Do you have document to explain both?

Reply:
I will add in the Design doc, the option fips is taken by linux kernel option, we are not ready for it now, so use sonic_fips instead.

Reply:
See https://github.com/sonic-net/SONiC/pull/997